### DISCUSSION TITLE
Cocoapods dependencies issue

### DISCUSSION THREAD

**Discussion by hchaucha** [capawesome-team/capacitor-firebase#431] [Q&A]
Hello,

I built a mobile application using **capacitor** (via _QuasarJS_) and **Firebase**.
I want to implement the authentication module with Google : it's working great on Android but I struggle to compile my project on IOS because of cocoapods dependencies.

It telling me that **capacitor-firebase/authentication** use the latest version of _Firebase_ (v.10.8.0) but the **CordovaPluginsStatic** using another version (v.9.3.0).

        In Podfile:
        CapacitorFirebaseAuthentication (from `../../node_modules/@capacitor-firebase/authentication`) was resolved to
        5.1.0, which depends on
        FirebaseAuth (= 10.8.0)
        
        CordovaPluginsStatic (from `../capacitor-cordova-ios-plugins`) was resolved to 5.4.0, which depends on
        Firebase/Auth (= 9.3.0) was resolved to 9.3.0, which depends on
        FirebaseAuth (~> 9.3.0)

What should I do ?
Should I downgrade/upgrade one of the two modules ?

I have already tried to downgrade firebase-capacitor but it seems that there is no compatible version with the targeted version of Firebase (+ its not a real/good solution, is it ?).

_Im aware that this is a known issue; not directly due to this module. Still, im not able to use it and no answer from the web help me_

Thanks for your time !

**Comment by robingenz** [comment]
Hey @hchaucha,
There must be a Cordova plugin in your project which uses the `Firebase/Auth` pod. You should uninstall that plugin. Capacitor Firebase should be the replacement. Feel free to share your `package.json`.
